Gemfile

```diff
@@ -50,6 +50,33 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
+group :test do
```

Comment:
Mmmm there are a lot of gems that you don't need for this task ( I suppose you added them because travis was failing because of rspec command)